scripts/ci/provisioning/freebsd.sh

```diff
@@ -29,6 +29,21 @@ if ! pkg info > /dev/null; then
   exit 1
 fi
 
+## Jan 7 2020 - work around image having mangled perl defaults
+if ! pkg info perl5.26; then
```

Comment:
this is still wrong, should be

```suggestion
if pkg info perl5.26; then
```